docs/source/quickstart/setup.mdx

```diff
@@ -125,7 +125,7 @@ type ReviewSummary {
   averageRating: Float
 }
 
-extend type Product @key(fields: "id") {
+type Product @key(fields: "id") {
   id: ID! @external
```

Comment:
The `@external` on `id` should be removed.

Reply:
Good point on both of these, yes, the quickstart is still using v1 schemas because it's pulling from the live deployed Acephei app that is _also_ still on Fed 1. We will need to update here, I can address next week